portal/models/intervention_strategies.py

```diff
@@ -182,7 +182,7 @@ def update_user_card_html(intervention, user):
             """.format(user.display_name)
             link_label = 'Begin questionnaire'
             link_url = url_for('assessment_engine_api.present_assessment',
-                               instrument_id='epic26')
+                               instrument_ids='epic26')
```

Comment:
Ah - I see.  You are actually parsing the comma delimited string.  This would introduce the third way for us to handle multiple values for a non-scalar.  In /api/coredata/acquire we take multiple 'require' parameters, which I thought was the preferred CGI approach for this.  Elsewhere, we use JSON arrays.  Can you convince me this approach is better?  I'd like our API to be as consistent as possible.